### ISSUE TITLE
Textareas in dialog boxes are not scrollable with the scroll wheel in Firefox

### ISSUE THREAD

**Issue by tmcw** [radix-ui/primitives#1417]
### Current Behavior

<img width="511" alt="CleanShot 2022-05-23 at 18 49 12@2x" src="https://user-images.githubusercontent.com/32314/169916892-a13f9b1d-f731-4b01-aee3-a19dbad11195.png">

Here's the demo (link below to the sandbox). It's the dialog demo from Radix's documentation, with an additional textarea with a lot of text in it.

On Chrome, you can use your mouse's scroll wheel or your touchpad two-finger scrolling to scroll the textarea. In Firefox, you can only scroll by dragging the scroll bar.

### Expected behavior

Scroll-wheel or two-finger scrolling should work in all browsers.

### Reproducible example

https://codesandbox.io/s/sad-pasteur-mg5d1i?file=/App.js

### Suggested solution

Maybe something is killing too many scroll events in the dialog?

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |      react-dialog   |  0.17       |
| React            | n/a     |    18.1.0     |
| Browser          |      Firefox   |     100.0.1 (64-bit)    |
| Operating System |     macOS    |    12.3.1     |

**Comment by benoitgrelard** [comment]
Hey @tmcw,

Thanks for reporting this, I can reproduce it.
That is very odd!! Especially given a regular scrolling div has no issues in Firefox, specifically a `textarea` does.
I suspect this has to do with one of the only dependency we use (for locking scroll), we'll take a look.

🙏

**Comment by innub** [comment]
Hey @benoitgrelard,

Your suspicions were right. After some digging, I found the culprit to be here:
https://github.com/theKashey/react-remove-scroll/blob/master/src/handleScroll.ts
![image](https://user-images.githubusercontent.com/76929931/174499569-75f2a13e-b0e6-4fa2-a6ec-5422941553d3.png)

On Chrome, when `overflowY` is either `unset` or `visible`, it counts as `auto` (no idea why). On Firefox `overflowY` defaults to `visible`. I found an old post that confirms this: https://bugzilla.mozilla.org/show_bug.cgi?id=984275

Until we a PR is made on their repo for a fix, an easy workaround would just be to set `overflow` to `auto`.

Here is a workaround demo: https://codesandbox.io/s/purple-hooks-hv3k2k?file=/App.js
